### ISSUE TITLE
Audio Manager behaviour

### ISSUE THREAD

**Issue by zrock-dev** [Pending-Name-21/screen#8]
> I have created a UML based on the current relationship- ![image](https://private-user-images.githubusercontent.com/64939916/343980816-a6f52f8e-a86f-41da-b5ce-527490f4ff86.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk1NDYwNDcsIm5iZiI6MTcxOTU0NTc0NywicGF0aCI6Ii82NDkzOTkxNi8zNDM5ODA4MTYtYTZmNTJmOGUtYTg2Zi00MWRhLWI1Y2UtNTI3NDkwZjRmZjg2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI4VDAzMzU0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzYzA5ZDRjYzkwMTk3MjcyMjM1MTlkMGEyZWU0OGY2NDZiOWZiMTY1ODIzZDM4NWYwYjM1YjhhOWFjNGU1MzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.X7udglTyCL62AJ4OOnj4l-HKMOoFzi-DWmxE2cAh4u0)
> 
> I must say that I find this relationship odd, AudioManager is meant to handle audio and Socket Server should provide it with the data. What do you think @codeFactory12? ![image](https://private-user-images.githubusercontent.com/64939916/343980920-bfa9de2c-026b-4866-bfdb-c703654b4fa9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk1NDYwNDcsIm5iZiI6MTcxOTU0NTc0NywicGF0aCI6Ii82NDkzOTkxNi8zNDM5ODA5MjAtYmZhOWRlMmMtMDI2Yi00ODY2LWJmZGItYzcwMzY1NGI0ZmE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI4VDAzMzU0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxMzZkNjlmZjMwODZkYTAzZWFiMTE2MmFiYmIyMTE0NTFlOGM2YmZhZDM1NDliMzA1ZDI1ZGIyYzcyZmZiNGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.goqg-TEOsoKuM75_z7AvycY1X50n6lQlV1uI4Jv5pIQ)

Yes it does but needs to be called the methods of  AudioManager manage all the logic to reproduce sounds bassed on a existent path, but needs to be called somewhere so can be able to be used

_Originally posted by @codeFactory12 in https://github.com/Pending-Name-21/screen/issues/5#issuecomment-2196094043_

**Comment by zrock-dev** [comment]
I agree, would it be okay for Audio Manager to receive only a path to perform audio management?

**Comment by codeFactory12** [comment]
? this issue report gives me no information about a bug.

**Comment by zrock-dev** [comment]
Dear @CodeFactory,

This issue was created in regard of the implementation of PR #12
handling the sound, this
is not a mention of a bug.

Sincerly,
Santiago C. Saavedra
